meetings/160427_organisers.md

```diff
@@ -11,4 +11,139 @@
+
+27 persons have already signed up.
+
+
+###SWC Workshop (20/21st June): 
+
+- Axel wish to teach too and asked if he could teach Lesson-2 and Lesson-3 for python. Hugues will teach the rest of the python lesson.
+
+- We also wish to give Michel the opportunity to teach (as part of his instructor training): Michel will teach part of the Unix shell.
+
+- Carrie and Anne will teach the R lessons.
+
+- Lex will teach git.
+
+- 14 persons signed up for this workshop. Only one signed up for one-day only.
+
+- Wide range of people from very different background. We were all very pleased about it!
+
+
+#### Actions ####
+
+- Carrie will send an email with the list of sub-lessons.
+- Lex will send an email to remind everybody to register: Early next week.
+- Anne will send emails and get more helpers!
+
+
+##post-workshop evaluation form for non-SWC/DC workshops##
+
+The R workshop in May is a non-SWC workshop.
+
+##improving the [workshop operations checklists](https://github.com/uio-carpentry/organisational/blob/master/workshop_operations/checklists.md)##
+
+We discussed the current steps to set-up a workshop at the University of Oslo. A recurrent problem is related to the access to the list of attendees.
+
+A way to solve it would be to get only one registration form (always active) and remove the persons when they attended a workshop. 
+
+####Actions####
+ 
+- Lex will set-up a meeting with the library to discuss (or through github).
+
+
+##Instructor Training workshop in Oslo June/July##
+
+An Instructor training workshop will be organized the first week of July in Oslo (and one in nederlands). 
+
+Christina Koch, Lex and Karin will be instructors.
+
+As part of our discussions, we all agreed to prioritize helpers and local people.
+Then it will be opened to a wider communities (all Norwegian, Nordic Universities)
+
+####Actions####
+
+- All: Try to find who could attend in our network.
+- All: we can all register to attend and refresh our knowledge!
+- Lex will send an email for the registration as soon as possible.
+
+##Fall workshop planning##
+
+Fall workshop planning (+ data therapy):
+
+- postponed it until June.
+
+As part of our discussions, we thought we could organize in fall a small seminar/presentation (about 45 minutes) to explain what we do, what workshops we have planned, etc.
+
+
+##meetups/software therapy/data therapy (‘beyond the workshop’)##
+
+Data therapy subgroup will meet on Friday.  
+
+Some important figures to take into account for the discussions:
+
+- What is the best way to exchange and discuss: discussion list or facebook, twitter, forum, etc.
+
+- we have about 30 instructors (organizers mailing list) that can potentially participate/teach/help to these Data Therapy.
+- We 242 people registered to our discussion mailing list.
+- So in total we have about 292 persons (attendees workshop + instructors)
+
+#### Actions ####
+- The summary and minutes of the subgroup meeting will be sent separately.
+
+##AOB##
```

Comment:
AOB means "Any Other Business". I used it here as the discussion John triggered was not in the agenda but still interesting to mention.